### PR TITLE
Unbreaks Rev

### PR DESCRIPTION
**Pull request by Incoming5643**
Somehow, someway, I forgot to have it return the list.

I am incoming, of infinite hotfixes.

### CODE REVIEW

File: code/game/gamemodes/game_mode.dm [hunk 1]
```diff
@@ -423,6 +423,7 @@
 	for(var/mob/living/carbon/human/player in mob_list)
 		if(player.stat!=2 && player.mind && (player.mind.assigned_role in security_positions))
 			sec |= player.mind
+	return sec
```
Comment (RemieRichards) [inline code/game/gamemodes/game_mode.dm]: these, and procs like them, should use ```.``` as the list, not a new var definition

Comment (duncathan) [inline code/game/gamemodes/game_mode.dm]: Making it `.` in the first place would have prevented this, too!

File: code/game/gamemodes/revolution/revolution.dm [hunk 2]
```diff
@@ -63,9 +63,6 @@
 	var/list/sec = get_living_sec()
 	var/weighted_score = min(max(round(heads.len - ((8 - sec.len) / 3)),1),max_headrevs)
 
-	if(weighted_score < 1)
-		weighted_score = 1
-
```
Comment (duncathan) [inline code/game/gamemodes/revolution/revolution.dm]: ?

Comment (Incoming5643) [inline code/game/gamemodes/revolution/revolution.dm]: It was a left over from the pull that introduced this bug. The max({formula},1) will do the same thing.

File: code/game/gamemodes/game_mode.dm (hunk 1 above)
Comment (Incoming5643) [inline code/game/gamemodes/game_mode.dm]: I do not know the secrets of the `.`

**Comment by Incoming5643** [comment]
Title unscrambled now that the round is over and PR meta has been defeated

File: code/game/gamemodes/game_mode.dm [hunk 3]
```diff
@@ -398,40 +398,38 @@
...
 //Keeps track of all living security members//
 //////////////////////////////////////////////
 /datum/game_mode/proc/get_living_sec()
-	var/list/sec = list()
+	. = list()
 	for(var/mob/living/carbon/human/player in mob_list)
-		if(player.stat!=2 && player.mind && (player.mind.assigned_role in security_positions))
-			sec |= player.mind
+		if(player.stat != DEAD && player.mind && (player.mind.assigned_role in security_positions))
+			. |= player.mind
```
Comment (Incoming5643) [inline code/game/gamemodes/game_mode.dm]: Let me know if this is proper form.

Comment (duncathan) [inline code/game/gamemodes/game_mode.dm]: Yes, this is correct.

Comment (RemieRichards) [inline code/game/gamemodes/game_mode.dm]: ya.

**Pull request merged**